tests/transformers/test_transformers_model_export.py

```diff
@@ -1362,6 +1363,52 @@ def test_text2text_generation_pipeline_with_inference_configs(
     pd_inference = pyfunc_loaded.predict(pd_input)
     assert pd_inference == result
 
+    model_path2 = tmp_path.joinpath("model2")
+    signature_with_params = infer_signature(
+        data,
+        mlflow.transformers.generate_signature_output(text2text_generation_pipeline, data),
+        inference_config,
+    )
+    mlflow.transformers.save_model(
+        text2text_generation_pipeline,
+        path=model_path2,
+        signature=signature_with_params,
+    )
+    pyfunc_loaded = mlflow.pyfunc.load_model(model_path2)
+
+    dict_inference = pyfunc_loaded.predict(
+        data,
+        params=inference_config,
+    )
+
+    assert dict_inference == inference
+
+
+@pytest.mark.skipif(RUNNING_IN_GITHUB_ACTIONS, reason=GITHUB_ACTIONS_SKIP_REASON)
```

Comment:
Can we remove the github actions skip decorators from the models that are testing the params functionality so that we can have daily visibility of any potential new library version release issues from transformers?